README.md

```diff
@@ -52,6 +49,19 @@ yarn report test/AddChannel.test.tsx     // run a single test instead of all
 ## ENV file setup
 
 In order to setup the environment variables correctly, you should create a .env file and inside copy the contents of the .env.example file.
-REACT_APP_API_URL - base API url that will be used when creating RTK Query queries
-REACT_APP_AUTH_TOKEN - authentication token required for access to certain endpoints
-REACT_APP_AUTH_METHOD - see https://github.com/Quansight/conda-store-ui/issues/53
+
+- `REACT_APP_API_URL` - base API url that will be used when creating RTK Query queries
+- `REACT_APP_AUTH_METHOD` - method of authentication.
+  - value `cookie` lets lets users authenticate with a login process. **This is the prefered option. conda-store API and conda-store UI must both be under the same domain.**
```

Comment:
Do you have one more "lets" word here? 
Precisely I mean:
_value `cookie` lets lets users authenticate..._ part

Reply:
lets lets lets fix this :)